pyanaconda/ui/tui/spokes/__init__.py

```diff
@@ -54,7 +54,10 @@ def __init__(self, data, storage, payload):
         Spoke.__init__(self, storage, payload)
 
         self.input_required = True
-        self.title = N_("Default spoke title")
+
+    @property
+    def title(self):
+        return _("Default spoke title")
```

Comment:
I don't think that we want to have TUI spokes with this default titles.

Reply:
OK, let's return `None` here as well.